post-installation/wpa_supplicant.conf

```diff
@@ -0,0 +1,10 @@
+# Allow users in the 'wheel' group to control wpa_supplicant  
+ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=wheel
+
+# Make this file writable for wpa_gui / wpa_cli
+update_config=1
+
+network={
+       ssid="<BSSID>"
+       psk="<PASSWORD>"
+}
```

Comment:
This is probably better embedded in the page itself. Keeps things nice and self-contained.